src/main/pages/che-7/end-user-guide/proc_defining-maven-settings-xml-file-across-workspaces.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="proc_defining-maven-settings-xml-file-across-workspaces_{context}"]
+= Defining Maven `settings.xml` file across workspaces
+
+To specify your own `settings.xml` file across workspaces, use the Secret with `settings.xml` and define labels and annotation to mount it as a file (see link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[Mounting a secret as a file into a workspace container]).
```

Comment:
>To specify your own `settings.xml` file across workspaces

May be here we can describe why it's might be needed and why it should be K8s secret and not ConfigMap

Reply:
> May be here we can describe why it's might be needed 

You mean why one might want to define own settings.xml ? I assume if someone find this doc, he know what he want's to achieve with maven, just don't know how to do it in Che.

> why it should be K8s secret and not ConfigMap

do we support mounting ConfigMap as a file ?

Reply:
No,  I mean add the sentence that describes that the main purpose is to be able to define credentials in settings.xml

Reply:
you're right. I rewrote the doc with repo creds example